src/Tasks.UnitTests/XmlPoke_Tests.cs

```diff
@@ -167,11 +179,11 @@ public void PokeMissingParams()
 
                 if (i < 7)
                 {
-                    Assert.True(exceptionThrown); // "Expecting argumentnullexception for the first 7 tests"
+                    exceptionThrown.ShouldBeTrue(); // "Expecting argumentnullexception for the first 7 tests"
```

Comment:
This looks cleaner to me for the whole block:

```suggestion
                if (i < 7)
                {
                    Should.Throw<ArgumentNullException>(() => p.Execute());
                }
                else
                {
                    Should.NotThrow(() => p.Execute());
                }
```

Reply:
Done!